docs/developers/omnichain/tutorials/single-input-multiple-output.md

```diff
@@ -0,0 +1,56 @@
+---
+sidebar_position: 5
+---
+
+# Single Input Multiple Output
+
+## Overview
+
+If you already read the previous tutorials you already know how to use zEVM. A
+very common use case on zEVM is a smart contract with a single input from one
+chain, do some magic, and then execute the output to another or multiple chains.
```

Comment:
maybe `do some logic` -> `perform some logic`

And we could add a sentence like `This capability may be useful for applications like multichain asset managers or DeFi applications that need to distribute or manage assets on many chains from one place.`